language/documentation/book/src/friends.md

```diff
@@ -0,0 +1,101 @@
+# Friends
+
+The `friend` syntax is used to declare modules that are trusted by the current module.
+A trusted module is allowed to call any function defined in the current module that have the `public(friend)` visibility.
+For details on function visibilities, please refer to the *Visibility* section in [Functions](./functions.md).
+
+## Friend declaration
+
+A module can declare other modules as friends via friend declaration statements, in the format of
+
+- `friend <address::name>` — friend declaration using fully qualified module name, or
+- `friend <module-name-alias>` — friend declaration using a module name alias, where the module alias is introduced via the `use` statement.
+
+A module may have multiple friend declarations, and the union of all the friend modules forms the friend list.
```

Comment:
Is there an example of a legal multiple friend declaration in your docs here? I didn't see it.